central/values.yaml

```diff
@@ -6370,7 +6366,7 @@ centraleventprocessor:
                   },
                   "rdkafkaConf": {
                     "client.id": "cep-con-notification-event",
-                    "group.id": "notifier",
+                    "group.id": "cep",
```

Comment:
Please align the group IDs to the standard naming conventions that we use elsewhere:
E.g.
                 "client.id": "cl-con-transfer-prepare",
                 "group.id": "cl-group-transfer-prepare",

I see you have done this for the client.id but not group.id.

Perhaps use:  `group.id": "cl-group-cep"`